zend_abstract_interface/methods/php5/methods.c

```diff
@@ -76,27 +74,77 @@ static bool z_call_method_without_args_ex(zval *object, zend_class_entry *ce, co
      */
     if (func->common.fn_flags & ZEND_ACC_ABSTRACT) {
         // "Cannot call abstract method %s::%s()"
-        return false;
+        goto error_exit;
+    }
+
+    zend_fcall_info_cache fcc = {0};
+    fcc.initialized = 1;
+    fcc.function_handler = func;
+    fcc.calling_scope = ce;
+    if (!object && (EG(called_scope) && instanceof_function(EG(called_scope), ce TSRMLS_CC))) {
```

Comment:
do we actually want to late static bind here? I'm not sure whether this is appropriate here, given the possibly arbitrary locations this zai method may be invoked from?

Reply:
I'm not sure. Can you think of a specific example of how LSB might cause an issue here?

Reply:
It doesn't cause an issue until it does :-D

Generally the scenario is you are in a parent class, an instrumentation is randomly triggered, invoking a method on the, and the child class uses static:: in its method - normally the call would go to the child method, except, if by a lot of bad luck, the call is triggered while we are in the parent class - then it goes to the parent method.

I have absolutely no idea if this will ever be an issue, but seems safer to me. Because I cannot imagine a scenario where LSB would be expected.

Reply:
You bring up a good point. I've disabled LSB to air on the safe side.